instrumentation/play/play-ws/play-ws-2.1/javaagent/build.gradle.kts

```diff
@@ -41,15 +41,33 @@ dependencies {
   testInstrumentation(project(":instrumentation:netty:netty-4.1:javaagent"))
   testInstrumentation(project(":instrumentation:akka:akka-http-10.0:javaagent"))
   testInstrumentation(project(":instrumentation:akka:akka-actor-2.3:javaagent"))
+}
+
+val testLatestDeps = findProperty("testLatestDeps") as Boolean
 
-  latestDepTestLibrary("com.typesafe.play:play-ahc-ws-standalone_2.13:+")
+testing {
+  suites {
+    val latestDepTest by registering(JvmTestSuite::class) {
+      dependencies {
+        implementation("com.typesafe.play:play-ahc-ws-standalone_2.13:+")
+      }
+    }
+  }
 }
 
-if (findProperty("testLatestDeps") as Boolean) {
-  configurations {
-    // play-ws artifact name is different for regular and latest tests
-    testImplementation {
-      exclude("com.typesafe.play", "play-ahc-ws-standalone_$scalaVersion")
+tasks {
+  if (testLatestDeps) {
+    // disable regular test running and compiling tasks when latest dep test task is run
```

Comment:
comment is a bit misleading, as far as I can tell compile task is not disabled

Reply:
this is only disabling test module. why do i need disable compile task?